frontend/src/stores/product.js

```diff
@@ -52,6 +61,7 @@ export const useProductStore = defineStore('productStore', {
       try {
         const data = await get_request('products/');
         this.products = Array.isArray(data) ? data : []; // Ensure products is always an array
+        console.log(this.products)
```

Comment:
Remove this one, to show the `products` content you can use Vue extension in Chrome, there you can see the parent components, children component and pinia states

Reply:
Changed